Chat/Stage 1/src/chat/Main.java

```diff
@@ -1,5 +1,5 @@
 package chat;
-
+// Это всё попытка обмануть Сканер при вводе отрицательного значения в строковую переменную 
```

Comment:
```suggestion
// Это всё игрушки
```
Резонное замечание, коллега! Немедленно сообщу заказчику и подумаем как вам оплалитить проект!